selendroid-standalone/src/main/java/io/selendroid/standalone/android/AndroidSdk.java

```diff
@@ -230,4 +236,40 @@ public static void setAndroidSdkVersion(String androidSdkVersion) {
   public static void setBuildToolsVersion(String buildToolsVersion) {
     sBuildToolsVersion = buildToolsVersion;
   }
+  public static File avdManager() {
+    if (sAvdManager != null) {
+      return new File(sAvdManager + platformExecutableSuffixExe());
+    }
+    else {
+      return android();
+    }
+  }
+
+  public static int getAndroidVersionNumber() {
+    if (sAndroidSdkVersion != null) {
+      try {
+        return Integer.parseInt(sAndroidSdkVersion);
+      } catch (NumberFormatException ex) {
+        log.warning("The android sdk version seems to be invalid: " + sAndroidSdkVersion);
+      }
+
+    }
+    if (androidSdkFolder() != null) {
+      String versionString = androidSdkFolder().getName();
+      log.info("Android SDK folder name is: " + versionString);
+      Pattern p = Pattern.compile(PLATFORM_VERSION_REGEX);
+      Matcher matcher = p.matcher(versionString);
+      if (matcher.matches()) {
+        return Integer.parseInt(matcher.group(1));
+      } else {
+        log.warning("Could not parse Android SDK version number " +
+                "from the sdk folder name !");
+      }
+    }
+    return -1;
```

Comment:
I'd prefer throwing in case of an error, returning an invalid value will lead to some hard to debug issues